packages/opds-web-client/webpack.common.js

```diff
@@ -0,0 +1,24 @@
+const path = require("path");
+const CleanWebpackPlugin = require('clean-webpack-plugin');
+var webpack = require('webpack');
+
+module.exports = {
+  plugins: [
+    new CleanWebpackPlugin(),
+    // jsdom is needed for server rendering, but causes errors
+    // in the browser even if it is never used, so we ignore it:
+    new webpack.IgnorePlugin(/jsdom$/)
+  ],
+
+  module: {
+    rules: [
+      {
+        test: /\.(ttf|woff|eot|svg|png|woff2|gif|jpg)(\?[\s\S]+)?$/,
+        loader: 'url-loader?limit=100000'
+      }
+    ]
+  },
+  resolve: {
+    extensions: [".webpack.js", ".web.js", ".js", ".ts", ".tsx", ".scss"]
```

Comment:
I know you just moved this from another file but we could probably take out the first two of these.